package/boot/uboot-mediatek/patches/438-add-xiaomi_mi-router-ax3000t.patch

```diff
@@ -0,0 +1,451 @@
+--- /dev/null
```

Comment:
Patch 438 already exists.

Reply:
Yep, 2 weeks ago 438 was latest. Renamed to 440